### PR TITLE
Added ability to see items history under admin - inside a dream to th…

### PR DESCRIPTION
**Pull request by rootux**
…e side there is a new versionate panel with abiliy to jump to older versions of the model

Also - ActiveAdmin uses the plural version of models - so camps instead of camp.
the permit params were not used and needed.

### CODE REVIEW

File: app/admin/camps.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  member_action :show do
+      @camp = Camp.includes(versions: :item).find(params[:id])
+      @versions = @camp.versions 
+      @camp = @camp.versions[params[:version].to_i].reify if params[:version]
+      show!
+  end
+
+  member_action :history do
+    @camp = Camp.find(params[:id])
+    @versions = PaperTrail::Version.where(item_type: 'Camp', item_id: @camp.id)
```
Comment (sh0oki) [inline app/admin/camps.rb:11]: camp.versions :)

File: app/views/layouts/history.html.arb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+panel "History" do
+  table_for assigns[:versions] do
+    column ("Item") do |v| 
+        if v.item
+           link_to v.item.id, [:admin, v.item]
+        end
+    end
+    column ("Type") { |v| v.item_type.underscore.humanize }
+    column ("Modified at") { |v| v.created_at.to_s :long }
+    column ("Text") { |v| v.object.to_s }
```
Comment (sh0oki) [inline app/views/layouts/history.html.arb:10]: arb! wow.
If diff is needed (not sure, but sounds useful), `v.changeset` will do that (if you add a `object_changes` column to help it).

Comment (rootux) [reply app/views/layouts/history.html.arb:10]: Interesting. it is a nice to have. just having the ability to go through history like this is already fun and useful.

**Comment by rootux** [comment]
@sh0oki So can I merge?

File: app/admin/camps.rb (hunk 1 above)
Comment (rootux) [reply app/admin/camps.rb:11]: in the original doc it said
    # @versions = @post.versions # <-- Sadly, versions aren't available in this scope, so use:
https://github.com/activeadmin/activeadmin/wiki/Auditing-via-paper_trail-(change-history)#adding-a-history-page

**Comment by sh0oki** [comment]
@rootux yeah

**Pull request merged**